src/flypad-backend/src/FlyPadBackend.h

```diff
@@ -29,12 +29,34 @@ using namespace std;
 // IDs for data structures - must be mapped to data structs
 enum DataStructureIDs {
   SimulationDataID,
-  PushbackDataID
+  PushbackDataID,
+  SelcalIVAODataID,
+  VolumeCOM1DataID,
+  VolumeCOM2DataID,
+  AllIVAODataID,
+  AircraftLoadedDataID,
+  SelcalVPILOTDataID,
+  AllVPILOTDataID
```

Comment:
Try to group these IDs if possible - similar to PushbackDataID - see in the cpp file.

If not possible some commenting would be great to clearly see they belong to the Selcal/ThridParty.